base/testdata/check_certs.go

```diff
@@ -0,0 +1,20 @@
+package main
```

Comment:
Yeah I have a script for this over in minikube. I'll see if it's easily portable over to here.